source/whats-new.txt

```diff
@@ -32,6 +32,16 @@ Learn what's new in:
 
 .. _upcoming-breaking-changes:
 
+.. _csharp-version-3.0:
```

Comment:
Can you add a link to this at the top, under the "Learn what's new in:" section?

Reply:
ah good call